airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -661,6 +661,7 @@ def stream_slices(
         parent_stream_slices = super().stream_slices(sync_mode=sync_mode, cursor_field=cursor_field, stream_state=parent_state)
         for parent_stream_slice in parent_stream_slices:
             yield {
+                "parent_updated_at": parent_stream_slice["parent"]["updated_at"],
```

Comment:
idea - maybe 'pull_request_updated_at' can be better ,  it will be similar to another attrs like 'pull_request_number'